core/testcasecontroller/algorithm/paradigm/lifelong_learning/lifelong_learning.py

```diff
@@ -0,0 +1,229 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Lifelong Learning Paradigm"""
+
+import os
+import shutil
+
+import numpy as np
+from sedna.datasources import BaseDataSource
+
+from core.common.constant import ParadigmType, SystemMetricType
+from core.testcasecontroller.algorithm.paradigm.base import ParadigmBase
+from core.testcasecontroller.metrics import get_metric_func
+from core.common.utils import get_file_format, is_local_dir
+
+
+class LifelongLearning(ParadigmBase):
+    # pylint: disable=too-many-locals
+    """
+    LifelongLearning
+    provide the flow of lifelong learning paradigm.
+    Notes:
+          1. Ianvs serves as testing tools for test objects, e.g., algorithms.
+          2. Ianvs does NOT include code directly on test object.
+          3. Algorithms serve as typical test objects in Ianvs
+          and detailed algorithms are thus NOT included in this Ianvs python file.
+          4. As for the details of example test objects, e.g., algorithms,
+          please refer to third party packages in Ianvs example.
+          For example, AI workflow and interface pls refer to sedna
+          (sedna docs: https://sedna.readthedocs.io/en/latest/api/lib/index.html),
+          and module implementation pls refer to `examples' test algorithms`,
+          e.g., basemodel.py, hard_example_mining.py.
+
+    Parameters
+    ---------
+    workspace: string
+        the output required for lifelong learning paradigm.
+    kwargs: dict
+        config required for the test process of lifelong learning paradigm,
+        e.g.: algorithm modules, dataset, initial model, incremental rounds,
+              model eval config, etc.
+
+    """
+
+    def __init__(self, workspace, **kwargs):
+        ParadigmBase.__init__(self, workspace, **kwargs)
+
+        self.incremental_learning_data_setting = kwargs.get("lifelong_learning_data_setting")
+        self.initial_model = kwargs.get("initial_model_url")
+
+        self.incremental_rounds = kwargs.get("incremental_rounds", 1)
+        self.model_eval_config = kwargs.get("model_eval")
+
+        self.cloud_task_index = '/tmp/cloud_task/index.pkl'
+        self.edge_task_index = '/tmp/edge_task/index.pkl'
+
+        self.system_metric_info = {SystemMetricType.SAMPLES_TRANSFER_RATIO.value: []}
+
+    def run(self):
+        # pylint:disable=duplicate-code
+        """
+        run the test flow of incremental learning paradigm.
+
+        Returns
+        ------
+        test result: numpy.ndarray
+        system metric info: dict
+            information needed to compute system metrics.
+
+        """
+
+        rounds = self.incremental_rounds
+        samples_transfer_ratio_info = self.system_metric_info.get(
+            SystemMetricType.SAMPLES_TRANSFER_RATIO.value)
+        #------my skip the training part----------
```

Comment:
The skip of training part is for testing but would infect other contributors' work. At least here an if-switch is needed instead of directly removing the code.